### PR TITLE
Update installation steps for Fedora 35

### PR DESCRIPTION
**Pull request by lsm5**
Mention how to fetch podman v4 on fedora 35 along with upgrade
scenario from f35 to f36

Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

@TomSweeneyRedHat @rhatdan @baude PTAL

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: getting-started/installation.md [hunk 1]
```diff
@@ -85,6 +85,20 @@ sudo apt-get -y install podman
 sudo dnf -y install podman
 ```
 
+Podman v4 will [not be
```
Comment (TomSweeneyRedHat) [inline getting-started/installation.md]:
```suggestion
**Podman v4** will [not be
```
Just to make it stand out a bit more.

File: getting-started/installation.md [hunk 2]
```diff
@@ -85,6 +85,20 @@ sudo apt-get -y install podman
...
+
+```bash
+sudo dnf -y copr enable rhcontainerbot/podman4
+sudo dnf -y install podman
+```
+
+This COPR repo should not cause any issues while upgrading from Fedora 35 to
+Fedora 36. You can simply delete the copr once you complete the upgrade.
+
+
```
Comment (TomSweeneyRedHat) [inline getting-started/installation.md]: rm extra line

File: getting-started/installation.md [hunk 3]
```diff
@@ -85,6 +85,20 @@ sudo apt-get -y install podman
 sudo dnf -y install podman
 ```
 
+Podman v4 will [not be
+shipped](https://podman.io/blogs/2022/03/06/why_no_podman4_f35.html) officially on Fedora 35.
+However, you can still fetch Podman v4 through our COPR repo.
```
Comment (TomSweeneyRedHat) [inline getting-started/installation.md]:
```suggestion
However, you can still fetch Podman v4 through our [Copr](https://copr.fedorainfracloud.org/) repo.
```

File: getting-started/installation.md [hunk 4]
```diff
@@ -85,6 +85,20 @@ sudo apt-get -y install podman
...
+
+```bash
+sudo dnf -y copr enable rhcontainerbot/podman4
+sudo dnf -y install podman
+```
+
+This COPR repo should not cause any issues while upgrading from Fedora 35 to
+Fedora 36. You can simply delete the copr once you complete the upgrade.
```
Comment (TomSweeneyRedHat) [inline getting-started/installation.md]: You have "COPR" on line 90 and "copr" here.  I think they should both be "Copr".  Also, do you have a quick pointer on how to do the delete?   Maybe add something like this (if correct):

For instance:

```bash
sudo dnf -y copr disable rhcontainerbot/podman4
```

Comment (lsm5) [reply getting-started/installation.md]: switched to using COPR across the board cause that's what I've seen used on some fedora docs.

**Comment by lsm5** [comment]
@TomSweeneyRedHat comments addressed PTAL.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**